wmn-data.json

```diff
@@ -5177,6 +5177,16 @@
         "known" : ["spicerex", "rod-it"],
         "cat" : "tech"
        },
+      {
+        "name" : "SPOJ",
+        "uri_check" : "https://www.spoj.com/users/{account}/",
+        "e_code" : 200,
+        "e_string" : "<li>Profile</li>",
+        "m_string" : " was not found on this server.",

```

Comment:
Not seeing this `m_string`. Consider something like `<strong>Innopolis Open 2018</strong>`